Fixed upstream, so backport to 0.17.4 of
https://github.com/aide/aide/commit/5161886c49060228811eee6da319844ef62dff6d
https://github.com/aide/aide/commit/007eb4f32e8874ce457b4f15d9789c2a9d61731b

Fixed in master but not 0.17.x.
--- a/configure.ac
+++ b/configure.ac
@@ -329,7 +329,7 @@ fi
 AC_CHECK_HEADERS(syslog.h inttypes.h fcntl.h ctype.h)
 
 PCRELIB="-lpcre"
-if test "$aide_static_choice" == "yes"; then
+if test "$aide_static_choice" = "yes"; then
     PCRELIB="$PCRELIB -pthread"
 fi
 AC_CHECK_LIB(pcre, pcre_exec, [
@@ -426,14 +426,14 @@ AS_IF([test "x$with_selinux_support" != xno],
     AC_DEFINE(WITH_SELINUX,1,[use SELinux])
     [AC_MSG_RESULT(yes)
     if test "x$PKG_CONFIG" != xno && $PKG_CONFIG --exists libselinux; then
-        if test "$aide_static_choice" == "yes"; then
+        if test "$aide_static_choice" = "yes"; then
             SELINUXLIB=$(${PKG_CONFIG} --libs libselinux --static)
         else
             SELINUXLIB=$(${PKG_CONFIG} --libs libselinux)
         fi
     else
     SELINUXLIB="-lselinux"
-    if test "$aide_static_choice" == "yes"; then
+    if test "$aide_static_choice" = "yes"; then
         saveLIBS=$LIBS
         LIBS="-static $SELINUXLIB"
         AC_SEARCH_LIBS([lgetfilecon_raw], [], [], [SELINUXLIB="$SELINUXLIB -lpthread"])
@@ -555,7 +555,7 @@ AC_ARG_ENABLE([default_db],
     [do not set default values for database_in and database_out config options]),
     [enable_default_db=$enableval],[enable_default_db=yes])
 
-if test "$enable_default_db" == "yes"; then
+if test "$enable_default_db" = "yes"; then
 
 if test "x$sysconfdir" != x'${prefix}/etc'; then
 	evalled_sysconfdir=`eval echo "$sysconfdir"`
@@ -592,7 +592,7 @@ AC_ARG_WITH([curl],
   [use curl library for http, https and ftp database backend (default: no)]),
  [with_curl=$withval], [with_curl=no])
 AS_IF([test "x$with_curl" = "xyes"], [
-       if test "$aide_static_choice" == "yes"; then
+       if test "$aide_static_choice" = "yes"; then
            PKG_CHECK_MODULES_STATIC(CURL, [libcurl], , [AC_MSG_RESULT([libcurl not found by pkg-config - Try --without-curl or add directory containing libcurl.pc to PKG_CONFIG_PATH environment variable])])
         else
            PKG_CHECK_MODULES(CURL, [libcurl], , [AC_MSG_RESULT([libcurl not found by pkg-config - Try --without-curl or add directory containing libcurl.pc to PKG_CONFIG_PATH environment variable])])
@@ -671,7 +671,7 @@ AS_IF([test "x$with_audit" != xno],
 		AC_MSG_ERROR(You don't have libaudit properly installed. Install it if you need it.)
 	)
 	AUDITLIB="-laudit"
-	if test "$aide_static_choice" == "yes"; then
+	if test "$aide_static_choice" = "yes"; then
 		saveLIBS=$LIBS
 		LIBS="-static $AUDITLIB"
 		AC_CHECK_LIB([audit], [audit_log_user_message], [], [
